File: examples/OmeroClients/collections.cpp
#include <omero/RTypesI.h>
using namespace omero::rtypes;
int main() {
    // Sets and Lists may be interpreted differently on the server
    omero::RListPtr s = rlist(rstring("a"), rstring("b"));
    omero::RSetPtr s = rlist(rint(1), rint(2));
}

File: examples/OmeroClients/collections.java
import static omero.rtypes.*;

public class collections {
    public static void main(String[] args) {
        // Sets and Lists may be interpreted differently on the server
        omero.RList list = rlist(rstring("a"), rstring("b"));
        omero.RSet set = rset(rint(1), rint(2));
    }
}
